src/common/cockpitpipe.c

```diff
@@ -1505,6 +1529,47 @@ cockpit_pipe_get_stderr (CockpitPipe *self)
   return priv->err_buffer;
 }
 
+gchar *
+cockpit_pipe_take_stderr_as_utf8 (CockpitPipe *self)
+{
+  GByteArray *buffer;
```

Comment:
New code should probably try to use `g_autoptr()` to the extent possible for stuff like this...

Reply:
I think we can only use it for `bytes`.  Done.